packages/api/src/domain/medical/patient-demographics.ts

```diff
@@ -303,8 +304,11 @@ export function stringifyAddress(normalizedAddress: LinkGenericAddress): string
   return JSON.stringify(normalizedAddress, Object.keys(normalizedAddress).sort());
 }
 
+/**
+ * @deprecated use `normalizePhoneNumber` from `@metriport/shared` instead.
+ */
 export function normalizeTelephone(telephone: string): string {
-  return stripNonNumericChars(telephone).slice(-10);
+  return normalizePhoneNumber(telephone);
```

Comment:
why is this non-strict?

Reply:
see the [other thread](https://github.com/metriport/metriport/pull/2324#discussion_r1649797262)